### ISSUE TITLE
Component with expand possibility

### ISSUE THREAD

**Issue by anders-kiaer** [equinor/webviz-core-components#176]
User request:

> In the surface viewer and the [structural uncertainty viewer](https://webviz-subsurface-example.azurewebsites.net/structural-uncertainty) I have a bit of a problem making the maps big enough to output to a decent screenshot. It would be awesome if we could maximize the map view somehow.

Should we make a `wcc` component which could facilitate expanding single "elements"/divs in a dashboard? Or perhaps better: Expand the current [`Frame` component](https://github.com/equinor/webviz-core-components/blob/a6ce00fe76d71cf4c11b2826639af47bb4511256/webviz_core_components/wrapped_components/frame.py#L6) with new argument (e.g. `allow_expansion=True/False`).

---

At a later stage maybe we want to allow the user to reorder these components within a dashboard/plugin, but this has not been scoped/adressed yet, but could be in "back of the mind" if relevant when implementing this feature.

**Comment by anders-kiaer** [comment]
Maybe parts of the `WebvizPluginPlaceholder` code can be reused?

If each div has a constantly visbile toolbar (like `WebvizPluginPlaceholder` currently has), we will use quite a lot of white space only on "div menu/buttons". Maybe we could switch to something visible only on hove in e.g. top right of divs? Could also consider doing the same change for the `WebvizPluginPlaceholder` as the menu there is currently using quite some white space compared to how small the  menu is (and how rarely you interact with the menu).

![image](https://user-images.githubusercontent.com/31612826/136588150-e8264983-4d1d-4a62-9b09-5faa7d28d590.png)

**Comment by anders-kiaer** [comment]
Another idea from today's standup meeting: It would be useful in "expanded mode" for a given card that plugin settings are still easily available in some way (for e.g. flipping through a map).

**Comment by rubenthoms** [comment]
This use case might be covered in the new Webviz Layout Framework (#219)?

**Comment by anders-kiaer** [comment]
I agree 🎉 The only thinkg that is not covered yet is this one perhaps: 
> Another idea from today's standup meeting: It would be useful in "expanded mode" for a given card that plugin settings are still easily available in some way (for e.g. flipping through a map).

Should we create a new separate issue for that idea and close this one @rubenthoms?
